sympy/simplify/simplify.py

```diff
@@ -522,6 +522,10 @@ def simplify(expr, ratio=1.7, measure=count_ops, fu=False):
         return expr
 
     if not isinstance(expr, (Add, Mul, Pow, ExpBase)):
+        if isinstance(expr, Function) and hasattr(expr, "inverse"):
+            if isinstance(expr.args[0], expr.inverse()):
```

Comment:
@asmeurer Should I pass `argindex=0` to `inverse` as we are using `expr.args[0]` ?